packages/tjdashboard/site/views/dashboard/tmpl/default.php

```diff
@@ -21,18 +21,29 @@
 $document->addScript('plugins/tjdashboardrenderer/tabulator/assets/js/jquery-ui.min.js');
 $document->addScript('plugins/tjdashboardrenderer/tabulator/assets/js/tabulator.min.js');
 $document->addStylesheet('plugins/tjdashboardrenderer/tabulator/assets/css/tabulator_semantic-ui.min.css');
+$document->addStylesheet('components/com_tjdashboard/assets/css/dashboard.css');
+$document->addStylesheet('plugins/tjdashboardrenderer/countbox/assets/css/countbox.css');
+$document->addStylesheet('media/techjoomla_strapper/css/bootstrap.j3.min.css');
+
+
 $document->addScript('plugins/tjdashboardrenderer/tabulator/assets/js/renderer.js');
 $document->addScript('components/com_tjdashboard/assets/js/tjDashboardService.js');
 $document->addScript('components/com_tjdashboard/assets/js/tjDashboardUI.js');
 
+$document->addScript('https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.min.js');
```

Comment:
@monikaatvowel This should be loaded by renderer.

Reply:
renderer js dynamically loaded now